pages/api/signup.js

```diff
@@ -1,21 +1,29 @@
 const { User } = require('../../models');
+const twilio = require('../../lib/twilio');
 
 export default async (req, res) => {
   if (req.method === 'POST') {
-    // generates random 4 character long alphanumeric string
-    req.body.verification_code = 'soft';
-
-    // insert into DB here
-    User.create(req.body)
-      .then((response) => res.json(response))
-      .catch((err) =>
-        res.status(409).json({
-          httpCode: 409,
-          message: 'User with this phone already exists',
+    if (!req.body.phone || req.body.phone.charAt(0) !== '+') {
+      res.status(400).json({
+        httpCode: 400,
+        message: 'Must provide valid phone number including + and country code',
+      });
+    } else {
+      // insert into DB here
+      User.create(req.body)
```

Comment:
This might a PR for a later time but I would prefer if you could be explicit about the parameters we are ingesting here. Something like: `{name: req.body.name, phone: req.body.phone}`